README.md

```diff
@@ -49,16 +49,14 @@ Short explanation: This generates the VTable layout for IUnknown and implements
 Interaction with COM components are always through an Interface Pointer (a pointer to a pointer to a VTable). We represent such an Interface Pointer with the `com::ComRc` struct, which helps manage the lifetime of the COM component through IUnknown methods.
 
 ```rust
-use com::runtime::ApartmentThreadedRuntime as Runtime;
-
 // Initialises the COM library
-let runtime = Runtime::new().expect("Failed to initialize COM Library");
+new_runtime::new(ApartmentThreaded).expect("Failed to initialize COM Library");
```

Comment:
Is it better to set the examples to use the Single Thread Apartment? Would that be more correct?